src/GitHubExtension/Widgets/GitHubReviewWidget.cs

```diff
@@ -1,219 +1,63 @@
-    {
-        // Remove event handler.
-        GitHubSearchManager.OnResultsAvailable -= SearchManagerResultsAvailableHandler;
-        base.DeleteWidget(widgetId, customState);
-    }
-
-    public override void OnWidgetContextChanged(WidgetContextChangedArgs contextChangedArgs)
-    {
-        Enabled = contextChangedArgs.WidgetContext.IsActive;
-        UpdateActivityState();
-    }
-
-    public new void UpdateActivityState()
-    {
-        // State logic for the Widget: (no configuration is needed)
-        // Signed in -> Active / Inactive per widget host.
-        if (!IsUserLoggedIn())
-        {
-            SetSignIn();
-            return;
-        }
-
-        ConfigurationState = "Activated";
-
-        if (Enabled)
-        {
-            if (ContentData == EmptyJson)
-            {
-                SetLoading();
-            }
-            else
-            {
-                SetActive();
-            }
-
-            return;
-        }
-
-        SetInactive();
+        // This widget doest not allow customization, so this value will not change.
+        ShowCategory = SearchCategory.PullRequests;
     }
 
     public override void RequestContentData()
     {
-        // Throttle protection against a widget trigging rapid data updates.
-        if (DateTime.Now - LastUpdated < WidgetDataRequestMinTime)
-        {
-            Log.Logger()?.ReportDebug(Name, ShortId, "Data request too soon, skipping.");
-        }
+        SearchIssuesRequest request = new SearchIssuesRequest($"review-requested:{UserName}");
 
-        if (ActivityState == WidgetActivityState.Configure)
-        {
-            return;
-        }
-
-        try
-        {
-            Log.Logger()?.ReportInfo(Name, ShortId, $"Requesting search for mentioned user {referredName}");
-            var requestOptions = new RequestOptions
-            {
-                ApiOptions = new ApiOptions
-                {
-                    PageSize = 10,
-                    PageCount = 1,
-                    StartPage = 1,
-                },
-                UsePublicClientAsFallback = true,
-            };
-
-            SearchIssuesRequest request = new SearchIssuesRequest($"review-requested:{ReferredName}");
-            var searchManager = GitHubSearchManager.CreateInstance();
-            searchManager?.SearchForGitHubIssuesOrPRs(request, Name, SearchCategory.PullRequests, requestOptions);
-            Log.Logger()?.ReportInfo(Name, ShortId, $"Requested search for {referredName}");
-            DataState = WidgetDataState.Requested;
-        }
-        catch (Exception ex)
-        {
-            Log.Logger()?.ReportError(Name, ShortId, "Failed requesting search.", ex);
-        }
+        RequestContentData(request);
     }
-
-    public override void LoadContentData()
-    {
-        var issuesData = new JsonObject
+
+    protected override string GetTitleIconData()
+    {
+        return IconLoader.GetIconAsBase64("pulls.png");

```

Comment:
Nit: Can use a more descriptive name. "ReviewWidgetTitleIcon.png"

Reply:
We don't have the actual icon for this widget. It is sharing the same icon with the Pull Requests widget.

Reply:
Ah I see. Might still be worth remaning to PullRequestsWidgetTitleIcon